### ISSUE TITLE
Pharo8/New Model: suggest simplest methods first

### ISSUE THREAD

**Issue by myroslavarm** [myroslavarm/Experimental-Completion#52]
for example when typing in 'init' the suggestions that appear don't include 'initialize' whereas it's actually the one most likely to be used and should appear at the top of the list

**Comment by MarcusDenker** [comment]
yes, sorting the results is very interesting... we need to add (next week) a sort API so that we can start to play with different sort strategies.

**Comment by myroslavarm** [comment]
the alphabetical sorter implemented here helps : https://github.com/myroslavarm/Experimental-Completion/commit/6d5da0dc753c571825ffb510ac9c9683d27b82fd but does not seem to solve the issue completely. question : do we keep this open or close it?
